File: lista2sala/1.py
n1=float(input("digite o valor da nota 1:"))
n2=float(input("digite o valor da nota 2:"))
n3=float(input("digite o valor da nota 3:"))
m=(n1+n2+n3)/3
if m>=6:
    print("aluno APROVADO com média:", '%.2f' %m)
else:
    print("aluno REPROVADO com média:", '%.2f' %m)

File: lista2sala/1.c
#include <stdio.h>
int main(){
    int n1, n2, n3, m;
    printf("digite a nota 1, 2 e 3:");
    scanf("%d%d%d", &n1, &n2, &n3 );
    m=(n1+n2+n3)/3;
    if (m>=6){
        printf("aluno aprovado %d", m);
    }
    else{
        printf("aluno reprovado %d", m);
    }
}
